src/util.mjs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export const randId = () => Math.random().toString(36).slice(2)
+
+export const isStr = (obj) => typeof obj === 'string'
```

Comment:
Also, let's make a func and rename it to isString.

Reply:
randId → randomId()

Btw, I can change impl to smth like to produce a bit more random randoms:
```js
export const randomId = (() => {
  const buffer = Buffer.alloc(10)
  return () => crypto.randomFillSync(buffer).toString('hex')
})()
``

Reply:
Let’s not bloat zx api any more =).

Reply:
If this only needed in tests, lets move it there.

Reply:
`randomId` is also used for script loader.